### ISSUE TITLE
Two issues with "mympd -c"

### ISSUE THREAD

**Issue by MichaIng** [jcorporation/myMPD#529]
myMPD version: 8.0.0

**Describe the bug**

1. When running `mympd -c`, it always returns with exit code 1, while not error is shown and all config files have been created successfully. This makes proper error-handling difficult.
2. Furthermore, the `MYMPD_LOGLEVEL` environment variable does not work as expected. When setting it to any valid value from 0-7, it always ends up with `/var/lib/mympd/config/loglevel` containing `0`. Only when it's not set, it contains the expected default `5`. But it has no impact on the exit code, which remains `1` in call cases.

**To Reproduce**

1. ```sh
    rm -Rf /var/lib/mympd
    export MYMPD_LOGLEVEL=5
    mympd -c
    echo $?
    cat /var/lib/mympd/config/loglevel
    ```

**Expected behavior**

`mympd -c` is expected to return the exit code `0` when not facing an error, otherwise should print an error message. `MYMPD_LOGLEVEL` is expected to be respected by `mympd -c` to be applied to `/var/lib/mympd/config/loglevel`, when valid, instead of `0`.

**Server plattform (please complete the following information):**
 - OS: DietPi on Debian Bullseye
 - OS version v7.4.-1 development version
 - Plattform: x86_64
 - MPD version: 0.22.6

**Comment by jcorporation** [comment]
Thanks for reporting.

The loglevel bug is already addressed in devel.

Error code bug will be fixed for 8.0.1

**Comment by MichaIng** [comment]
Ah, I should have a look into devel commits. Many thanks 👍.
